packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -434,281 +287,205 @@ class ClientJWKSResult(PangeaResponseResult):
-    pass
+class FlowUpdateDataProfile(APIRequestModel):
+    profile: Profile
 
 
-# - path: authn::/v1/flow/verify/social
-# https://pangea.cloud/docs/api/authn#social-sign-in
-class FlowVerifySocialRequest(APIRequestModel):
-    flow_id: str
-    cb_state: str
-    cb_code: str
+class FlowUpdateDataProvisionalEnrollment(APIRequestModel):
+    state: str
+    code: str
 
 
-class FlowVerifySocialResult(CommonFlowResult):
-    pass
+class FlowUpdateDataResetPassword(APIRequestModel):
+    # FIXME: review
+    state: str
+    code: str
 
 
-# - path: authn::/v1/user/mfa/delete
-# https://pangea.cloud/docs/api/authn#delete-mfa-enrollment
-class UserMFADeleteRequest(APIRequestModel):
-    user_id: str
-    mfa_provider: MFAProvider
+class FlowUpdateDataSetEmail(APIRequestModel):
+    email: str
 
 
-class UserMFADeleteResult(PangeaResponseResult):
-    pass
+class FlowUpdateDataSetPassword(APIRequestModel):
+    password: str
 
 
-# - path: authn::/v1/user/mfa/enroll
-# https://pangea.cloud/docs/api/authn#enroll-in-mfa
-class UserMFAEnrollRequest(APIRequestModel):
-    user_id: str
-    mfa_provider: MFAProvider
+class FlowUpdateDataSMSOTP(APIRequestModel):
     code: str
 
 
-class UserMFAEnrollResult(PangeaResponseResult):
-    pass
-
-
-# - path: authn::/v1/user/mfa/start
-# https://pangea.cloud/docs/api/authn#start-mfa-verification
-class UserMFAStartRequest(APIRequestModel):
-    user_id: str
-    mfa_provider: MFAProvider
-    enroll: Optional[bool] = None
-    phone: Optional[str] = None
+class FlowUpdateDataSocialProvider(APIRequestModel):
+    social_provider: str
+    uri: str
 
 
-class UserMFAStartTOTPSecret:
-    qr_image: str
-    secret: str
+class FlowUpdateDataTOTP(APIRequestModel):
+    code: str
 
 
-class UserMFAStartResult(PangeaResponseResult):
-    totp_secret: Optional[UserMFAStartTOTPSecret] = None
+class FlowUpdateDataVerifyEmail(APIRequestModel):
+    state: str
+    code: str
 
 
-# - path: authn::/v1/user/mfa/verify
-# https://pangea.cloud/docs/api/authn#verify-an-mfa-code
-class UserMFAverifyRequest(APIRequestModel):
-    user_id: str
-    mfa_provider: MFAProvider
-    code: str
+FlowUpdateData = Union[
+    Dict,
+    FlowUpdateDataAgreements,
+    FlowUpdateDataCaptcha,
+    FlowUpdateDataEmailOTP,
+    FlowUpdateDataMagiclink,
+    FlowUpdateDataPasskey,
+    FlowUpdateDataPassword,
+    FlowUpdateDataProfile,
+    FlowUpdateDataProvisionalEnrollment,
+    FlowUpdateDataResetPassword,
+    FlowUpdateDataSetEmail,
+    FlowUpdateDataSetPassword,
+    FlowUpdateDataSMSOTP,
+    FlowUpdateDataSocialProvider,
+    FlowUpdateDataTOTP,
+    FlowUpdateDataVerifyEmail,
+]
+
+
+class FlowUpdateRequest(APIRequestModel):
+    flow_id: str
+    choice: FlowChoice
+    data: FlowUpdateData
 
 
-class UserMFAVerifyResult(PangeaResponseResult):
+class FlowUpdateResult(CommonFlowResult):
     pass
 
 
-# - path: authn::/v1/user/verify
-# https://pangea.cloud/docs/api/authn#verify-user
 class UserVerifyRequest(APIRequestModel):
```

Comment:
the /v1/user/verify endpoint is private.

Reply:
Yes, it was removed in next commit